contracts/contracts/token/OUSD.sol

```diff
@@ -40,23 +67,41 @@ contract OUSD is Governable {
 
     uint256[154] private _gap; // Slots to align with deployed contract
     uint256 private constant MAX_SUPPLY = type(uint128).max;
+    /// @dev The value of tokens in existence
     uint256 public totalSupply;
     mapping(address => mapping(address => uint256)) private allowances;
+    /// @dev Address of the vault with privileges to execute {mint},
```

Comment:
Probably don't need "Address of" since its not only the type but in the name as well.

Reply:
gotcha